homeworks/vlad.makedon_Vladmaked/homework_6/user-info.js

```diff
@@ -0,0 +1,29 @@
+const user = {};
+
+Object.defineProperties(user, {
+    fullName: {
+        get: () => this.fullName,
```

Comment:
arrow function will mutate global scope here, `this` becomes a reference to global object